### PR TITLE
Changelog for 0.3.1

### PR DESCRIPTION
**Pull request by ax3l**
Changelog for `0.3.1`:

- add changelog, summary and contributions
- bump version number to `0.3.1`

note: compile suite has changed, manual compile test in #2304 on hypnos was successful

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,61 @@
...
+   - cmakeFlags: escape lists #2183
+   - tbg:
+     - `--help` exit with 0 return code #2213
+     - env variables: proper handling of \ and & #2262
+
+**Misc:**
+ - PIConGPU: `--help` to stdout #2148
+ - tools: all to C++11 #2194
+ - documentation:
+   - Hypnos .tpl files: mpiexec `LD_LIBRARY_PATH` #2149
```
Comment (PrometheusPi) [inline CHANGELOG.md]: OPTIONAL:
The meaning of this brief description is not clear to me. (But the reference should be enough - so it is fine with me).

Comment (ax3l) [reply CHANGELOG.md]: what it does: it updates the tbg .tpl files for the hypnos queues. 
do you have a suggestion how to improve it?

Comment (ax3l) [reply CHANGELOG.md]: I could also write:
`- tbg: Hypnos .tpl files: remove warnings #2149`

Comment (PrometheusPi) [reply CHANGELOG.md]: Do you like:
```
Hypnos .tpl files: remove passing LD_LIBRARY_PATH to avoid warning
```
?

**Comment by ax3l** [comment]
@ComputationalRadiationPhysics/picongpu-maintainers please also check the announcement text, this is the moment to adjust it before it goes in meta data and mail announcements.

**Comment by ax3l** [comment]
@ComputationalRadiationPhysics/picongpu-developers @ComputationalRadiationPhysics/picongpu-maintainers feel free to give the `release-0.3.1` branch also some runtime tests :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,61 @@
 Change Log / Release Log for PIConGPU
 ================================================================
 
+0.3.1
+-----
+**Date:** 2017-10-11
+
+Field Energy Plugin, Gaussian Density Profile and Restarts
+
+This release fixes the energy field plugin diagnostics (not all field
```
Comment (psychocoderHPC) [inline CHANGELOG.md]: The energy plugin accessed wrong memory within the stack. The resulting error can not be estimated.

Maybe we should remove the description with `()`

**Comment by PrometheusPi** [comment]
@ax3l You wrote that the manual compile test was successful. Can we merge now?

**Comment by PrometheusPi** [comment]
ping @ax3l: Please confirm that the current version `77edff4` pasted the compile test.

**Comment by ax3l** [comment]
before we merge this, we will add an other RC to the release branch with two more fixes.

Added a [WIP] to the title.

**Comment by ax3l** [comment]
added the rc5 changes to the changelog, will rebase after #2312 is merged

**Comment by ax3l** [comment]
Since I tested the PR in #2312 as a compile test and this only add the change log the compile test should be fine.

**Pull request merged**